ui/article-renderer/src/main/java/org/cru/godtools/article/activity/ArticlesActivity.java

```diff
@@ -0,0 +1,58 @@
+package org.cru.godtools.article.activity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.FragmentManager;
+
+import org.cru.godtools.article.fragment.ArticlesFragment;
+import org.cru.godtools.base.tool.activity.BaseSingleToolActivity;
+import org.cru.godtools.article.R;
+import java.util.Locale;
+
+public class ArticlesActivity extends BaseSingleToolActivity {
+
+    private static final String CATEGORY_KEY = "category_key";
+    private String mCategoryID;
+
+    //region Initialization
+    public static void start(@NonNull final Context context, @NonNull final String toolCode,
+                                @NonNull final Locale language, @NonNull final String categoryID) {
+        Bundle args = new Bundle();
+        populateExtras(args, toolCode, language);
+        args.putString(CATEGORY_KEY, categoryID);
+        Intent intent = new Intent(context, ArticlesActivity.class).putExtras(args);
+        context.startActivity(intent);
+    }
+
+    public ArticlesActivity() {
+        super(true);
```

Comment:
I was opting to not make any of the articles UI immersive, so make this `super(false)`